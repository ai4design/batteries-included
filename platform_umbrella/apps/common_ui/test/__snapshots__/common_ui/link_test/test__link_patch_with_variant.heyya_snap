<a href="/test" data-phx-link="patch" data-phx-link-state="push" class="font-medium text-pink-500 hover:text-pink-600 hover:underline">
  Test patch with var
</a>