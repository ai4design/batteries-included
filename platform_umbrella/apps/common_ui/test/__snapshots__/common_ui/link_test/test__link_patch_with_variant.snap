"<a href=\"/test\" data-phx-link=\"patch\" data-phx-link-state=\"push\" class=\"font-medium text-secondary-500 hover:text-secondary-600 hover:underline\">\n  Test patch with var\n</a>"